### ISSUE TITLE
can't download cert

### ISSUE THREAD

**Issue by KazumiLine** [ButterCam/Mediator#5]
i can't download the cret, when i click download, it only shows the html page

**Comment by devkanro** [comment]
It is a known issue, please visit cert link direct.

Mediator provide multi format of Root Certificate, you can choose the format you like.
```
http://<YOUR PC/MAC IP>:8888/mediatorRoot.cer
http://<YOUR PC/MAC IP>:8888/mediatorRoot.crt
http://<YOUR PC/MAC IP>:8888/mediatorRoot.pem
```

**Comment by devkanro** [comment]
Fixed in 1.4.4

**Comment by lebanggit** [comment]
Still error: Unauthorized Access
